### PR TITLE
Get restaurant mentions

### PR DESCRIPTION
**Pull request by h314to**

### CODE REVIEW

**Comment by coppetti** [comment]
why get twitter mentions are placed  on "list members extractors"?

File: src/main/scala/org/sf1bacon/twitterextractor/Restaurant.scala [hunk 1]
```diff
@@ -17,22 +17,71 @@ case class Restaurant(name: String,
...
-         ON MATCH SET r.followers = ${this.followers},
-                      r.tweets = ${this.tweets},
-                      r.verified = ${this.verified},
-                      r.location = "${this.location}",
-                      r.url = "${this.url}",
-                      r.updated = "${Utils.timestamp}"
-        """.stripMargin)
+
+  def queryRestaurant(modtime: String): String =
+    s"""MERGE (r:Restaurant { id: "${this.username}", name: "${this.name}", username: "${this.username}" })
```
Comment (coppetti) [inline src/main/scala/org/sf1bacon/twitterextractor/Restaurant.scala]: could remove username: same info of id

File: src/main/scala/org/sf1bacon/twitterextractor/Restaurant.scala [hunk 2]
```diff
@@ -17,22 +17,71 @@ case class Restaurant(name: String,
...
+                     r.verified = ${this.verified},
+                     r.location = "${this.location}",
+                     r.url = "${this.url}",
+                     r.updated = "$modtime"
+    """.stripMargin
+
+  def queryMentions(t: Tweet) : String =
+    s"""MERGE (u:User {id: "${t.user.get.screen_name}",
+                       name: "${t.user.get.name}",
+                       username: "${t.user.get.screen_name}"})
```
Comment (coppetti) [inline src/main/scala/org/sf1bacon/twitterextractor/Restaurant.scala]: remove username here in case you remove on previous step

**Review by coppetti — CHANGES_REQUESTED** [review]

**Review by m-maga — APPROVED** [review]

**Review by JVieira13 — APPROVED** [review]
Mais uma baconzada :)

**Comment by h314to** [comment]
@coppetti : ok, done

**Pull request closed without merging**